### ISSUE TITLE
Cronicle upgrade doesn't work with a custom installation path

### ISSUE THREAD

**Issue by 1992kk** [jhuckaby/Cronicle#778]
## Summary

Cronicle upgrade doesn't work with a custom installation path. We've installed cronicle at a custom path "/path/cronicle". However, when /path/cronicle/bin/control.sh upgrade, it initiates a fresh install in /opt/cronicle directory.

## Steps to reproduce the problem

- Manually install cronicle at /path/cronicle
- To upgrade it run /path/cronicle/bin/control.sh upgrade
- This will not upgrade cronicle in /path/cronicle but will do a fresh installation in /opt/cronicle.

### Operating system and version?

Rocky linux 8.7

### Node.js version?

v16.20.2

### Cronicle software version?

0.9.45

### Are you using a multi-server setup, or just a single server?
1 Master and 3 workers

### Are you using the filesystem as back-end storage, or S3/Couchbase?
Filesystem

### Can you reproduce the crash consistently?
Yes

**Comment by jhuckaby** [comment]
Yeah, Cronicle **really** wants to live in `/opt/cronicle`.  There are a bunch of things that expect that base path, particularly the `control.sh` script and all the CLI commands.

I'm really sorry you ran into this.  I'll add a louder mention of this in the docs.
